go.mod

```diff
@@ -0,0 +1,23 @@
+module github.com/mendersoftware/mender
+
+require (
+	github.com/bmatsuo/lmdb-go v0.0.0-20160816100615-69ad631904c9
+	github.com/davecgh/go-spew v1.1.0
+	github.com/konsorten/go-windows-terminal-sequences v1.0.2
+	github.com/mendersoftware/log v0.0.0-20180403084710-7fef0b7a1659
+	github.com/mendersoftware/mender-artifact v0.0.0-20191118115643-7c541c8f8348
+	github.com/mendersoftware/mendertesting v0.0.0-20190301095837-dedd2c0a90a4
+	github.com/mendersoftware/scopestack v0.0.0-20180403075023-2ce74757611b
+	github.com/pkg/errors v0.0.0-20160916110212-a887431f7f6e
+	github.com/pmezard/go-difflib v1.0.0
+	github.com/remyoudompheng/go-liblzma v0.0.0-20190506200333-81bf2d431b96
+	github.com/sirupsen/logrus v0.0.0-20180329225952-778f2e774c72
+	github.com/stretchr/objx v0.1.0
+	github.com/stretchr/testify v0.0.0-20190109162356-363ebb24d041
+	github.com/ungerik/go-sysfs v0.0.0-20190613143942-7f098ddb67a6
+	github.com/urfave/cli v1.21.0
```

Comment:
Also, this is wrong, as we are using our own `urfave/cli` fork, but with the regular `urfave/cli` import path. That's what tripped the migration off here.

Reply:
thanks for pointing that out! I was not aware that this would be in there... should now be fixed 👌